src/main/java/net/sf/jabref/event/AddEntryEvent.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+public class AddEntryEvent {
+
+    private BibEntry bibEntry;
+
+    public AddEntryEvent(BibEntry bibEntry) {
+        setBibEntry(bibEntry);
+    }
+
+    public void setBibEntry(BibEntry bibEntry) {
```

Comment:
The idea was to leave possibility for setting `bibEntry` later. I can imagine that updating the `bibEntry` to another (within one event) is needed. But this situation didn't occur yet, so I'm going to remove it.